### ISSUE TITLE
Measurement summary

### ISSUE THREAD

**Issue by tilsche** [tud-zih-energy/lo2s#31]
Show a short summary after execution. It is suppressed with `-q`.

Possible things to display
  * [x] Number of spawned threads/processes during execution
 * [x] Name and arguments of executed binary
* [x] Execution wall time / CPU time
 * [x] Name of generated trace file
 * [x] Size of generated trace file
 * [x] Number of wakeups

Maybe this is already too much, we need to keep it concise.

**Comment by cvonelm** [comment]
My suggestion for the output format: 

```bash
EXECUTION SUMMARY
 * executed command: bash 
 * 8.75s Wall Time
 * 0.09s CPU time
 * spawned threads: 2
 * wakeups: 103
 * traces have been written to: lo2s_trace_2018-01-11T12-08-37(7766B)
```
Ideas for improvement(?):

- show the exit status of the command we've ran, e.g. "executed command: bash (-1)"
- Rewrite the file size as KiB or MiB once we reach a certain threshold e.g. traces larger than 2048 B -> write as 2KiB

**Comment by bmario** [comment]
I doubt that displaying the executed command has any merit. The user had to pass that as arguments anyways and thus should know that already.

Further ideas for things to show:
- [ ] in `-a` mode, the number of all different recorded processes might be interesting 

The actual format of the summary should be discussed in the PR.

**Comment by tilsche** [comment]
I would suggest a more compact layout along the lines of

```
[ lo2s: bash (0), 8.09s CPU, 8.72s total, 1 processes, 2 threads ]
[ lo2s: 123 wakeups, wrote 2.12 GiB lo2s_trace_2018-01-11T12-08-37 ]
```

We can discuss further refinement. First line is different for `-a`.

**Comment by bmario** [comment]
Fixed with 893d3b92cf36b63f9fe2e156b2806f89097399c2
